tests/test_stokes.py

```diff
@@ -45,22 +49,42 @@ def StokesTest(N, butcher_tableau, stage_type="deriv", splitting=AI):
 
     u, p = z.subfunctions
     u.interpolate(uexact)
-
-    lu = {"mat_type": "aij",
-          "snes_type": "newtonls",
-          "snes_linesearch_type": "l2",
-          "snes_linesearch_monitor": None,
-          "snes_monitor": None,
-          "snes_rtol": 1e-8,
-          "snes_atol": 1e-8,
-          "snes_force_iteration": 1,
-          "ksp_type": "preonly",
-          "pc_type": "lu",
-          "pc_factor_mat_solver_type": "mumps"}
+    p.interpolate(pexact-assemble(pexact*dx))
+
+    ns = butcher_tableau.num_stages
+    ind_pressure = ",".join([str(2*i+1) for i in range(ns)])
+    solver_params = {
+        "mat_type": "aij",
+        "snes_type": "ksponly",
+        "ksp_type": "fgmres",
+        "ksp_max_it": 200,
+        "ksp_gmres_restart": 30,
```

Comment:
Not critical for a test, but these seem like generic defaults.  Do we expect this to take more than O(10) iterations?  If not, why not set both `max_it` and `restart` to a smaller value like 30?